code/routing/FluentRootURLController.php

```diff
@@ -99,4 +99,22 @@ public function handleRequest(SS_HTTPRequest $request, DataModel $model = null)
 		$this->popCurrent();
 		return $result;
 	}
+    
+    /**
+     * 
+     * With Translatable installed, don't pre-append the locale to the homepage 
+     * URL.
+     * 
+     * @return string
+     * @see {@link Translatable::get_homepage_link()}.
+     */
+    public static function fluent_homepage_link($localeURL) {
+        $homepageLink = parent::get_homepage_link();
+        if(class_exists('Translatable')) {
```

Comment:
Is there ever a case where a user has translatable AND fluent, but wants to use the 'fluent home page' rather than the translatable one?

Should this be a configurable option that, by default (perhaps), acts as you have set above?

Really nice fix and looks good. :D